internal/pkg/server/fleet.go

```diff
@@ -47,26 +48,28 @@ const kUAFleetServer = "Fleet-Server"
 
 // Fleet is an instance of the fleet-server.
 type Fleet struct {
-	bi     build.Info
-	verCon version.Constraints
+	standAlone bool
+	bi         build.Info
+	verCon     version.Constraints
 
 	cfgCh    chan *config.Config
 	cache    cache.Cache
 	reporter state.Reporter
 }
 
 // NewFleet creates the actual fleet server service.
-func NewFleet(bi build.Info, reporter state.Reporter) (*Fleet, error) {
+func NewFleet(bi build.Info, reporter state.Reporter, standAlone bool) (*Fleet, error) {
```

Comment:
can we bind this to a flag rather than hardcoding passed values

Reply:
this is what i had in mind, keep parameter for testing but use a Fn implementing `return true/false` based on DEV flag as input to NewFleet.